pwa/src/pages/endpoints/[endpointId].tsx

```diff
@@ -51,7 +73,7 @@ export const IndexPage = (props) => {
             </div>
             <div className="tab-pane" id="logs" role="tabpanel" aria-labelledby="logs-tab">
               <br />
-              <LogTable {...{ endpointId }} />
+              <LogTable logs={logs} />
```

Comment:
Currently, the LogTable is rendering on a grey background and without the right layout (padding and margins and such). We can fix this, by adding a `Card` component around it (just like other tables have), like so:

```
<Card title="Endpoint Logs" cardHeader={() => <></>} cardBody={() => <LogTable logs={logs} />} />
```

You can also add a refresh button in the `cardHeader={() => ...}`, in the above example I'm just adding a empty object.

Note: this also goes for all the other files where you've implemented this!